docs/functions.rst

```diff
@@ -108,23 +108,32 @@ Example:
         - |
           echo "Example"
 
-KubeTask
+Dump
```

Comment:
are we removing KubeTask function in favour of this function? docs changes look like that but release note say new function is added.